scapy/autorun.py

```diff
@@ -78,6 +78,8 @@ def __init__(self):
             self.s = ""
         def write(self, x):
             self.s += x
+        def flush(self):
```

Comment:
That function was missing: multiprocessing.Process would crash (because for some reasons it needed to execute sys.stdout.flush()  )